tests/expand/pr32-in.jsonld

```diff
@@ -0,0 +1,13 @@
+{
+  "@context": [{
+    "@version": 1.1,
+    "id": {"@id": "@id", "@protected": true},
+    "type": {"@id" : "@type", "@container": "@set", "@protected" : true},
+    "@type": {"@container": "@set", "@protected": true}
+  }, {
+    "@version": 1.1,
+    "@type": {"@protected": true}
```

Comment:
Yes,
Because it would change the container association with `@type`. But, this might not be important, because it wouldn’t change the interpretation when expanding, but could create an alternate representation when compacting.

Reply:
+1 to @gkellogg -- and better not to have subtle surprises like that.